src/bot/scripts/card.js

```diff
@@ -473,12 +469,70 @@ const cb = async (client, message) => {
 
     .catch(console.error);
 
-  let cardData = cache.cachedCard[cardId].data;
+  // card select
+
+  let card = cache.cachedCard[cardId].data;
 
   let level = commandValues.value;
 
-  // card select
-  let card = cache.cachedCard[cardId].data;
+  let commandModifier = commandValues.modifier;
```

Comment:
You don't have to change this BUT next time use an object with keys and values for these (90% of the time a better option to switch)